### PR TITLE
feh: Add package option

### PR DESCRIPTION
**Pull request by Wisehh**
### Description

Resolve the request I made in: https://github.com/nix-community/home-manager/issues/3495

@rycee Thank you for the indications and I remain attentive to any problem!

### Checklist

- [x] Change is backwards compatible.

- [ ] Code formatted with `./format`.

- [ ] Code tested through `nix-shell --pure tests -A run.all`.

- [ ] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

  - [ ] Added myself and the module files to `.github/CODEOWNERS`.

### CODE REVIEW

File: modules/programs/feh.nix [hunk 1]
```diff
@@ -27,6 +27,13 @@ in {
   options.programs.feh = {
     enable = mkEnableOption "feh - a fast and light image viewer";
 
+    package = mkOption {
+      type = types.package;
+      default = pkgs.feh;
+      defaultText = literalExpression "pkgs.feh";
+      description = "The feh package to use";
+    };
```
Comment (ncfavier) [inline modules/programs/feh.nix:35]:
```suggestion
    package = mkPackageOption pkgs "feh";
```

**Review by sumnerevans — APPROVED** [review]

**Review by ncfavier — APPROVED** [review]

File: modules/programs/feh.nix [hunk 2]
```diff
@@ -27,6 +27,8 @@ in {
   options.programs.feh = {
     enable = mkEnableOption "feh - a fast and light image viewer";
 
+    package = mkPackageOption pkgs "feh";
```
Comment (ncfavier) [inline modules/programs/feh.nix:30]:
```suggestion
    package = mkPackageOption pkgs "feh" {};
```

Sorry

**Review by ncfavier — CHANGES_REQUESTED** [review]

File: modules/programs/feh.nix (hunk 2 above)
Comment (sumnerevans) [reply modules/programs/feh.nix:30]: probably needs the space to make the formatter work

```suggestion
    package = mkPackageOption pkgs "feh" { };
```

**Comment by Wisehh** [comment]
Is this okay?

**Review by ncfavier — APPROVED** [review]

**Pull request merged**